github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml

```diff
@@ -754,7 +754,7 @@ presubmits:
         env:
         - name: FEATURE_GATES
           value: NonRootExperimental
-        - name: NONROOT
+        - name: KUBEVIRT_NONROOT
```

Comment:
@jean-edouard Did you check if the release recognizes this?

Reply:
I assumed so, since the file already uses `KUBEVIRT_NONROOT` for all the other nonroot jobs.
I just checked and indeed, 0.47 and 0.48 do check for `KUBEVIRT_NONROOT` in `kubevirt/automation/test.sh`.